src/web/app/components/question-types/question-edit-details-form/mcq-question-edit-details-form.component.ts

```diff
@@ -16,9 +16,12 @@ export class McqQuestionEditDetailsFormComponent
     extends QuestionEditDetailsFormComponent<FeedbackMcqQuestionDetails> {
 
   readonly PARTICIPANT_TYPES: string[] = [FeedbackParticipantType.STUDENTS,
```

Comment:
while you are at it, maybe add line break before the first element in the array too?